laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js

```diff
@@ -0,0 +1,328 @@
+/**
+ * To add TinyMCE drop down for shortcode generator.
+ *
+ * @package laterpay
+ */
+/*global laterpay_shortcode_generator */
```

Comment:
```suggestion
/* globals laterpay_shortcode_generator, tinymce, wp */
```
This should be the first line.